### PR TITLE
Rest/Get Up Icon

### PR DESCRIPTION
**Pull request by Birdtalon**
Adds Rest/Rise icon for laying/crawling and getting up.

dreamseeker_2017-09-24_13-06-11

🆑 Birdtalon
add: Adds crawling / standing icon to the UI
/🆑

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -947,3 +947,15 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 			riding_datum.unequip_buckle_inhands(M)
 			riding_datum.restore_position(M)
 	. = ..(M, force)
+
+/mob/living/carbon/human/update_rest_icon(mob/living/M)
+	var/obj/screen/lay/byond_sucks
```
Comment (AndrewMontagne) [inline code/modules/mob/living/carbon/human/human.dm]: Can we rename this something more meaningful?

Comment (Birdtalon) [reply code/modules/mob/living/carbon/human/human.dm]: Byond sucks

**Comment by SamHPurp** [comment]
This icon is.... poor.

**Comment by AndrewMontagne** [comment]
[Crawling button](https://trello.com/c/vnaEOyRI/40-crawling-button)

**Comment by FlattestGuitar** [comment]
I gotta agree with Purpose. We'll need a proper icon for this that matches the style of the rest.

As we talked earlier, it'd be nice to squish it in along with the walk/run toggle. Either vartically to keep the pics or horizontally to just have the words `crawl`, `stand`, `walk`, `run` in there.

**Pull request merged**